app/indexers/identity_metadata_indexer.rb

```diff
@@ -41,10 +41,24 @@ def catkey
     @catkey ||= Array(cocina_object.identification.catalogLinks).find { |link| link.catalog == 'symphony' }&.catalogRecordId
   end
 
+  def previous_catkeys
+    @previous_catkeys ||=
+      Array(cocina_object.identification.catalogLinks).filter_map { |link| link.catalogRecordId if link.catalog == 'previous symphony' }
+  end
+
   def folio_instance_hrid
     @folio_instance_hrid ||= Array(cocina_object.identification.catalogLinks).find { |link| link.catalog == 'folio' }&.catalogRecordId
   end
 
+  def previous_folio_instance_hrids
+    @previous_folio_instance_hrids ||=
+      Array(cocina_object.identification.catalogLinks).filter_map { |link| link.catalogRecordId if link.catalog == 'previous folio' }
+  end
+
+  def previous_ils_ids
+    @previous_ils_ids ||= Array(previous_catkeys + previous_folio_instance_hrids)
```

Comment:
Since both `#previous_catkeys` and `#previous_folio_instance_hrids` both are guaranteed to return an array, I think the `Array()` here should not be necessary?

Reply:
I tried it without and it was unhappy.  :-(

Reply:
IIRC

Reply:
@ndushay As in, specs were failing? Or did the badness manifest in some other way?

Reply:
I was confused -- I got the error in `catkey` and `folio_instance_id` methods

Reply:
(so now fixed)

Reply:
🍻